feature/mpls/static/otg_tests/static_bgp_nexthop/README.md

```diff
@@ -0,0 +1,80 @@
+# MPLS-2.2: MPLS forwarding via static LSP to BGP next-hop.
+
+## Summary
+
+Validate static LSP functionality with BGP resolved next-hop.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_4.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed)
+
+## Procedure
+
+### Configuration
+
+1) Create the topology below:
+
+    ```
+                         |         | ---- | ATE Port 2 | ---- [eBGP peer]
+    [ ATE Port 1 ] ----  |   DUT   |      |            |
+                         |         | ---- | ATE Port 3 |
+    ```
+
+2)  Configure eBGP peer on ATE Port 2 interface and advertise BGP-NH-V4 and BGP-NH-V6
+3)  Configure discard static routes for BGP-NH-V4 and BGP-NH-V6 to Null0 via administrative distance of 254.
+4)  Enable MPLS forwarding.
+5)  Create egress static LSP for IPv4 traffic to pop the label and resolve the next-hop BGP-NH
+    *  Match incoming label (1000004)
+    *  Set the action to pop label
+    *  Set IPv4 next-hop and resolve NH action
+6)  Create egress static LSP for IPv6 traffic to pop the label and resolve the next-hop BGP-NH
+    *  Match incoming label (1000006)
+    *  Set the action to pop label
+    *  Set IPv4 next-hop and resolve NH action
+7)  Configure cover static routes to IPV4-DST and IPV6-DST to ATE Port 3.
```

Comment:
I am not sure what is meant by the "cover" static routes?  I think we need a concrete example, also in OC.  To help I will provide example static route OC:

```yaml
network-instances:
  - network-instance:
    protocols:
      - protocol:
        static-routes:
          - static:
            config:
              prefix: "some prefix here"
              set-tag: (if needed)
              description: "something describing why this static route is needed"
            next-hops:
              - next-hop:
                config:
                  index:  1
                  next-hop:  "ip, interface, etc" ref: https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-static-routes-static-next-hops-next-hop-config-next-hop
                  metric:  1
```

Reply:
Done